src/main/java/com/liferay/faces/util/logging/LoggerFactory.java

```diff
@@ -79,23 +70,12 @@
 	 */
 	public static final Logger getLogger(String name) {
 
-		Logger logger = null;
-
-		try {
```

Comment:
@ngriffin7a, seems like there is no need for a try/catch here since LoggerLog4JImpl swallows all exceptions: https://github.com/liferay/liferay-faces-util/blob/master/src/main/java/com/liferay/faces/util/logging/internal/LoggerLog4JImpl.java#L34-L64

Is that right?

I will test if you think this is a good change.